tendrl/node_agent/flows/generate_journal_mapping/__init__.py

```diff
@@ -0,0 +1,20 @@
+import json
+
+from tendrl.commons import flows
+from tendrl.commons.objects.job import Job
+from tendrl.node_agent.provisioner.ceph import utils
+
+
+class GenerateJournalMapping(flows.BaseFlow):
+    def run(self):
+        # Generate the journal mapping for the nodes
+        mapping = utils.generate_journal_mapping(
+            self.parameters['Cluster.node_configuration']
+        )
+
+        # Update output dict
+        job= Job(job_id=self.job_id).load()
+        output_dict = json.loads(job.output)
```

Comment:
Even after initializing the output using `dict(_None=None)` here the `job.output` is string `None` and doesnt allow to execute `ob.output[class name] = mapping`.

Not sure how this would work.